services/app-api/src/domain-models/contractAndRates/dataValidatorHelpers.ts

```diff
@@ -90,4 +92,82 @@ const validateContractDraftRevisionInput = (
     return parsedData.data
 }
 
-export { validateContractDraftRevisionInput }
+const parseContract = (
+    contract: ContractType,
+    stateCode: string,
+    store: Store,
+    featureFlags?: FeatureFlagSettings
+): ContractType | Error => {
+    const contractParser = featureFlags?.['438-attestation']
```

Comment:
Not blocking: I would like to see us add a pattern here instead of code right in the parser.  We know will have many more feature flag conditional fields on the backend. And that code will be temporary. 

Maybe a function that takes `submittableContractSchema` and the full list of feature flags as params and returns the enhanced schema